source/frontend/src/features/properties/map/MapContainer.tsx

```diff
@@ -45,14 +45,17 @@ const MapContainer: React.FC<React.PropsWithChildren<MapContainerProps>> = () =>
   );
 };
 
+// prettier-ignore
```

Comment:
We need to disable prettier here because we really really need double quotes below (single quotes will not do it). Given that our project is setup to have prettier use single quotes instead of double ones then we need to disable prettier for the next line